core/arch/aarch64/emit_utils.c

```diff
@@ -370,14 +370,14 @@ patch_branch(dr_isa_mode_t isa_mode, cache_pc branch_pc, cache_pc target_pc,
     ASSERT(ALIGNED(branch_pc, 4) && ALIGNED(target_pc, 4));
```

Comment:
>  i#2974 trace support for AArch64, part 18: Adding comment on preferring
linear control flow

Note that the ", part N: " is a convention for merges to master: i.e., this PR being merged will be part 1; the next PR for trace building merged into master would be part 2 (not part 19 or sthg: the PR-internal ones don't count toward the master merges).

Reply:
sure, that's why I put part 2 in my final commit message above